src/controllers/CommentsController.ts

```diff
@@ -0,0 +1,55 @@
+import { Request, Response } from 'express'
+import validateComment from '../validation/addComment'
+import { Comments, User } from '../db'
+import CustomError from '../helpers/CustomError'
+import verifyToken from '../helpers/verifyToken'
+import { IToken } from '../interfaces/IToken'
+
+export default class CommentsController {
+  // for getting all data
+  public static async index (req: Request, res: Response) {
+    const { eventId } = req.params
+    const allComments = await Comments.findAll({
+      where: { EventId: eventId },
+      include: {
+        model: User,
+        attributes: ['id', 'username', 'profileImg']
+      }
+    })
+    res.json({ message: 'Comments received successfully', data: allComments })
+  }
+
+  // for getting just on element of data
+  //  (like getting just one event may be in event details)
+  public static async show (req: Request, res: Response) {
+  // code here
+  }
+
+  // for storing new data
+  public static async store (req: Request, res: Response) {
```

Comment:
Then you may need to create a custom interface for request
```
interface IUserRequest extends Request {
  user?: IToken,
}
```

```suggestion
  public static async store (req: IUserRequest, res: Response) {
  const user = req.user as IToken; // verified
```